packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -191,15 +191,22 @@ const pincodeSetScreenOptions = ({
 }: {
   route: RouteProp<StackParamList, Screens.PincodeSet>
 }) => {
-  const isVerifying = route.params?.isVerifying
-  const title = isVerifying
-    ? i18n.t('onboarding:pincodeSet.verify')
+  const changePin = route.params?.changePin
```

Comment:
This is losing the `isVerifying` part. We could simplify by doing:
```
const isVerifying = route.params?.isVerifying
const changePin = route.params?.changePin
let title
if (isVerifying) title = i18n.t('onboarding:pincodeSet.verify')
else if (changePin) title = i18n.t('onboarding:pincodeSet.changePIN')
else title = i18n.t('onboarding:pincodeSet.create')
```
(this assumes that we want the title to be the verifying one the second time).

another solution is welcome too of course, but lets make sure we take into account the `verifying` thing.

Reply:
i took it out because in the designs, there actually isn't a difference between isVerifying and not... the title is always just Change PIN! figma here: 
![image](https://user-images.githubusercontent.com/12700801/114480073-9f115b00-9bb6-11eb-8467-4fc6a2cb4512.png)

Reply:
You're right! Seems like we got rid of the `isVerifying` thing.